subcommittees/mentoring/PracticumProposal.md

```diff
@@ -0,0 +1,196 @@
+# Instructor Training Practicum Proposal
+
+## Background and motivation
+
+There are several reasons to offer a follow-up training experience to the base instructor training.
+These reasons are: 
+
+First, many freshly minted instructors have stated that once they are out of teacher training, they
+still feel that they are not comfortable teaching our lessons. One of the things that can be done to
+alleviate this is to ensure that they have worked through the lesson material that they will be 
+teaching from, and also had experience in teaching from the material. Through this, we ensure that
+they know the lesson material, and have helped and guided them through what they will be teaching.
+It is likely that this will lower the threshold to teaching their first workshop.
+
+Second, we are expecting that the people we admit to be our teachers already know certain things.
+This might not be the case - we could have people who are great at Unix but who know little about
+Git, and people with R knowledge who would not be able to manoeuvre on the command line. Thus, there
+might be gaps in their knowledge, which the trainees themselves may or may not be aware of. By having
+them work through the core lesson material, we ensure that the trainees have at least a minimum level
+of knowledge in all of our core subjects. 
+
+Third, many instructors don’t really know too many others within the Software and Data Carpentry
+organizations. By having people working in teams during training, we strengthen between-instructor
+connections which may lead to more discussions between instructors at a later stage. Also, by having
+people already within the community mentoring future instructors, they get to know people already
+inside of the organization. 
+
+Fourth, as a bonus we get a lot of people working through the material, which can only be a benefit to
+the lesson development process. It might draw more people into helping out with lesson development, and
+could lower the bar for people developing new lessons.
+
+With these goals in mind, we have developed a post-instructor training practicum.  
+
+## Practicum elements
+
+The instructor training practicum aims to facilitate the transition from the introductory instructor
+training  to actually teaching workshops, by doing the following activities: 
+
+### Mentoring
+
+Future instructors, or trainees, form small groups and are assigned a mentor.  Over a period of a 3-4
```

Comment:
On Thu, Dec 03, 2015 at 10:14:44AM -0800, Raniere Silva wrote:
> > -1 to GMT+3, because it doesn't handle daylight savings time.
>
> Do you have one suggestion? I know that `tzselect` has a list of
> cities that we could use.

Greg's point about wanting airports later on [1] is sufficient
justification for me.  I'd explain that motivation in this proposal,
so folks knew why we weren't just asking for timezones.  There may be
cases where someone lives in one timezone, but their closest major
airport is in a different timezone, but I expect those will be very
rare.

[1]: https://github.com/swcarpentry/board/pull/65/files#r46582531